web/packages/teleterm/src/ui/ConnectMyComputer/permissions.ts

```diff
@@ -29,20 +27,16 @@ import * as tsh from 'teleterm/services/tshd/types';
  * https://github.com/gravitational/teleport/pull/28346#discussion_r1246653846
  * */
 export function hasConnectMyComputerPermissions(
-  rootCluster: Cluster,
+  loggedInUser: tsh.LoggedInUser,
```

Comment:
This change isn't entirely necessary, but it makes it easier to pass ACL both to this function and to `getEmptyTableStatus`.

The downside is that `hasConnectMyComputerPermissions` no longer checks for root cluster. So far this wasn't actually needed anywhere, but with this PR it actually is required – we're adding a Connect My Computer related UI in a component that is shown for both root and leaf clusters.